src/style.css

```
@@ -0,0 +1,31 @@
+* {
+  margin: 0;
+}
+
+.stars--container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars-star {
+  height: 16px;
+  width: 16px;
+
+  margin-right: 4px;
+
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 > .stars-star,
+.stars--1 :not(:first-child),
+.stars--2 > .stars-star:nth-child(n+3),
+.stars--3 > .stars-star:nth-child(n+4),
+.stars--4 :last-child {
```

You can also do it in this way (maybe will be more understandable)
```suggestion
.stars--0 > :nth-child(-n+1),
.stars--1 > :nth-child(-n+2),
.stars--2 > :nth-child(-n+3),
.stars--3 > :nth-child(-n+4),
.stars--4 > :nth-child(-n+5), {
```